dotnet/src/SemanticKernel.Abstractions/Filters/AutoFunctionInvocation/AutoFunctionInvocationContext.cs

```diff
@@ -19,21 +19,25 @@ public class AutoFunctionInvocationContext
     /// <param name="function">The <see cref="KernelFunction"/> with which this filter is associated.</param>
     /// <param name="result">The result of the function's invocation.</param>
     /// <param name="chatHistory">The chat history associated with automatic function invocation.</param>
+    /// <param name="chatMessageContent">The chat message content associated with automatic function invocation.</param>
     public AutoFunctionInvocationContext(
```

Comment:
Do we need to keep this public or could we use internals visible to?

Reply:
Discussed internally to keep it temporarily as `public` and hide it later when function calling logic will be extracted.